calypso/README.md

```diff
@@ -81,3 +94,14 @@ as a target an existing instance.
 The read contract verifies that the request is valid and points to the write
 instance. It stores the reader's public key in the instance, so that the
 secret-management cothority can re-encrypt to this reader's public key.
+
+## Resharing LTS
+It is possible that the roster might change and the LTS shares must be
+re-distributed but without changing the LTS itself. We accomplish this in two
+steps.
+
+1. The authorised client(s) must update the LTS roster in the blockchain (an
+   instance of the LTS smart contract).
+2. Then, the client instructs the calypso conodes to run the resharing
+   protocol. The members of the old calypso finds the new roster and then
+   executes the actual protocol.
```

Comment:
Can you propose what I should change it to?

Reply:
Instead of "The members of the old calypso finds the new roster and then
   executes the actual protocol", perhaps say "The nodes in the new roster finds and checks the the proof of roster-change in ByzCoin, and then starts the protocol to reshare the secret between themselves."